ide-support/revsaveasiosstandalone.livecodescript

```diff
@@ -998,6 +995,12 @@ private command revCopyMobileFiles pFiles, pBaseFolder, pAppBundle, pAppTarget,
          next repeat
       end if
       
+      -- If the file is an entitlement then handle appropriately
+      if tSource ends with ".xcent" then
+         put url ("binfile:" & tSource) after rCustomEntitlements
```

Comment:
Hmm... all the plists being loaded in this script are treated as binary and have been for a long time. They are utf-8 so would need to be read and decoded and then re-encoded to ensure they weren't treated as MacRoman and have their line endings changed. I would suggest leaving as is and adding to the backlog if it's necessary which I'm not convinced it is.